config/foreman-debug.conf

```diff
@@ -33,5 +33,38 @@
 # Permanently disable upload feature (0 or 1)
 #UPLOAD_DISABLED=0
 
+# URL of the upload location (string)
+#UPLOAD_URL='rsync://theforeman.org/debug-incoming'
+
+# The full upload command in strict quotes (string)
+#UPLOAD_CMD='rsync "${TARBALL}" "${UPLOAD_URL}"'
+
+# Additional help message for when uploads are not disabled (UPLOAD_DISABLED=0) (multi line string)
+#UPLOAD_USAGE=$( cat << UPLOAD_USAGE
+#You may want to upload the tarball (with -u) to our public server via rsync.
+#This is a write-only directory (readable only by Foreman core developers)
+#Please note that the rsync transmission is UNENCRYPTED.
+#UPLOAD_USAGE
+#)
+
+# Message displayed at the end if neither UPLOAD nor UPLOAD_DISABLED is set (multi line string)
+#UPLOAD_MSG_UNSET="To upload a tarball to our secure site, please use the -u option.\n"
+
+# Message when an upload was successfull (multi line string)
+# note: will be appended with "$(basename ${TARBALL})\n"
+#UPLOAD_MSG_SUCCESS=$( cat << UPLOAD_MSG_SUCCESS
+#The tarball has been uploaded, please contact us on our mailing list or IRC
+#referencing the following URL:\n
+#    http://debugs.theforeman.org/
+#UPLOAD_MSG_SUCCESS
+#)
+
+# Message when an upload was not successfull (multi line string)
+#UPLOAD_MSG_FAIL=$( cat << UPLOAD_MSG_FAIL
+#The tarball could not be uploaded, please upload it to an alternate location
+#and contact us on our mailing list or IRC referencing that URL.
+#UPLOAD_MSG_FAIL
+#)
```

Comment:
Can you name all "message" variables in a same pattern? I'd prefer `MSG_*`. Also, I don't think we need to have copies of those in comments, just put "Custom message goes here" or similar.

Reply:
I can do that. Will have a look at it tomorrow.